lib/index.js

```diff
@@ -53,22 +53,43 @@ $(document).ready(function(){
       $(".third").text(hourlyWeather[2]["icon"])
       $(".fourth").text(hourlyWeather[3]["icon"])
       $(".fifth").text(hourlyWeather[4]["icon"])
-      $(".first-t").text(hourlyWeather[0]["time"])
-      $(".second-t").text(hourlyWeather[1]["time"])
-      $(".third-t").text(hourlyWeather[2]["time"])
-      $(".fourth-t").text(hourlyWeather[3]["time"])
-      $(".fifth-t").text(hourlyWeather[4]["time"])
-      $(".first-temp").text(Math.ceil(hourlyWeather[0]["temperature"]))
-      $(".second-temp").text(Math.ceil(hourlyWeather[1]["temperature"]))
-      $(".third-temp").text(Math.ceil(hourlyWeather[2]["temperature"]))
-      $(".fourth-temp").text(Math.ceil(hourlyWeather[3]["temperature"]))
-      $(".fifth-temp").text(Math.ceil(hourlyWeather[4]["temperature"]))
-      // $.each(hourlyWeather, function (index, value) {
-      //   $(".hourly-info").html(`<td class="round-${index}">${value["icon"]}</td>`);
-      //   $(".hourly-info").html(`<td class="round-${index}">${value["time"]}</td>`);
-      //   $(".hourly-info").html(`<td class="round-${index}">${Math.ceil(value["temperature"]}°</td>`);
-      // });
-      
+      $(".first-t").html(`<b>${hourlyWeather[0]["time"]}</b>`)
+      $(".second-t").html(`<b>${hourlyWeather[1]["time"]}</b>`)
+      $(".third-t").html(`<b>${hourlyWeather[2]["time"]}</b>`)
+      $(".fourth-t").html(`<b>${hourlyWeather[3]["time"]}</b>`)
+      $(".fifth-t").html(`<b>${hourlyWeather[4]["time"]}</b>`)
+      $(".first-temp").text(`${Math.ceil(hourlyWeather[0]["temperature"])}°`)
```

Comment:
Instead of explicitly setting these class names with first, second, third, etc...Maybe look into dynamically generating these class names by looping over the hourlyWeather times. You can do this with the various other sets of data that is coming back from your API. This will clean up your code and make it more DRY.

Reply:
I created a waffle card for this! I spent some time today researching this and will need to do some more googling. Due to our check in tomorrow, my priority right now is getting the API calls to the various endpoints functioning and then go back and refactor!

Reply:
@mgoodhart5 You should always aim to get basic/foundational features hashed out first. It's always easier to improve something that works than try to update something that doesn't work quite yet!

Reply:
Thank you!!!